packages/@okta/vuepress-site/docs/guides/oin-oidc-guide/protocol-level-requirements/index.md

```diff
@@ -2,52 +2,98 @@
+* The browser receives an authorization code response from the Okta authorization server. If there is an error or the authentication fails for any reason, the browser returns the error to the end user. If the authentication is successful, the Okta authorization server issues an authorization code value and includes it in a redirect that sends the browser over to the Client at the Login Redirect URI callback location.
 * The Client (your application) then submits a request to the Okta `/token` endpoint using the authorization code value and the Client Secret. If both are valid, then Okta returns:
   * `access_token` &mdash; this token is restricted to give access only to what the end user authorized for your application
   * `token_type=Bearer` &mdash; this identifies the authentication method used
   * `expires_in` &mdash; the expiration time for the access token (in seconds)
   * `scope` &mdash; the scopes that are contained in the access token
   * `id_token` &mdash; the ID token contains the specific info (in JWT format) about the authentication and the end user that is needed by the application
   * Optional. `refresh_token` &mdash; this token is returned if the `offline_access` scope is granted. This token can be exchanged for a new access token if the previous access token has expired
-* The access token is then used by your application to retrieve the protected resources and information requested at the Resource Server for the end user
+* The access token is then used by your application to retrieve the protected resources and information requested at the Resource Server for the end user.
 
-The authorization code flow depends on Okta and your SaaS back-end systems being able to securely communicate the Client Secret.
+The Authorization Code flow depends on Okta and your SaaS back-end systems being able to securely communicate the Client Secret.
 
-Note that your application needs to verify user identity through a user store, otherwise your app could be spoofed by fake applications in the Android or Apple app stores, or an end user could view and possibly modify your code. For this reason, native, mobile, and single page application (SPA) integrations are not acceptable for OIDC app integrations in the OIN. Your application code must be set up in a flow so that your client application talks to your SaaS back-end which then communicates with Okta.
+Note that your application needs to verify user identity through a user store. Otherwise, your app could be spoofed by fake applications in the Android or Apple app stores, or an end user could view and possibly modify your code. For this reason, native and mobile app integrations are not acceptable for OIDC app integrations in the OIN. You must set up your application to use an authentication flow so that your client application talks to your SaaS back-end, which then communicates with Okta.
```

Comment:
```suggestion
> **Note:** Your application needs to verify user identity through a user store. Otherwise, your app could be spoofed by fake applications in the Android or Apple app stores, or an end user could view and possibly modify your code. For this reason, native and mobile app integrations aren't acceptable for OIDC app integrations in the OIN. You must set up your application to use an authentication flow so that your client application talks to your SaaS back end, which then communicates with Okta.
```